From bf05d0035dedbf6df44e303560c966b7354389e2 Mon Sep 17 00:00:00 2001
From: "Rich.jiangyk" <jyk@rock-chips.com>
Date: Mon, 13 Sep 2010 16:51:28 +0800
Subject: [PATCH] fix ov9650 init_data bug

in sensor_init_data array, register 0x00 was set to 0x00, which cause the initialization unfinished.
This setting is useless, so remove it! ov9650 is working better at vga dc/dv now.
---
 drivers/media/video/ov9650.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/video/ov9650.c b/drivers/media/video/ov9650.c
index 42b81b80b9bd..4930818e1c77 100644
--- a/drivers/media/video/ov9650.c
+++ b/drivers/media/video/ov9650.c
@@ -105,7 +105,7 @@ static struct reginfo sensor_init_data[] =
     {0x13, 0xe0},
     {0x01, 0x80},
     {0x02, 0x80},
-    {0x00, 0x00},
+//    {0x00, 0x00},	//jyk_0913, this setting is useless, but cause init unfinished! 
     {0x10, 0x00},
     {0x13, 0xe5},
     {0x39, 0x50},
-- 
2.35.3

